web/packages/teleport/src/Bots/InfoGuide.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * Teleport
+ * Copyright (C) 2025 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Box from 'design/Box';
+import { Mark } from 'design/Mark';
+
+import {
+  InfoExternalTextLink,
+  InfoParagraph,
+  InfoTitle,
+  ReferenceLinks,
+} from 'teleport/components/SlidingSidePanel/InfoGuideSidePanel';
+
+const InfoGuideReferenceLinks = {
+  Bots: {
+    title: 'What are Bots',
+    href: 'https://goteleport.com/docs/enroll-resources/machine-id/introduction/#bots',
+  },
+  TBots: {
```

Comment:
```suggestion
  TBot: {
```

tbot is the CLI, so I think it should be singular. (We would say "tsh", not "tshs")